docs/developers/packaging.md

```diff
@@ -82,35 +93,43 @@ supports the following installer types:
 * On Windows, a graphical installer based on NSIS is generated.
 
 The configuration is done through a `construct.yaml` file, documented [here][7]. We generate one on
-the fly in the `bundle_conda.py` script. Roughly, we will build this configuration file:
+the fly in the `build_installers.py` script found in `napari/packaging`.
+Roughly, we will build this configuration file:
 
 
 ```yaml
 # os-agnostic configuration
 name: napari
-version: 0.4.12  # depending on workflow triggers, this can also be a dev snapshot for nightlies
+version: "0.0.1"  # this is the _internal_ version of the installer infrastructure
 company: Napari
-license: <points to a generated license that bundles the licenses of all dependencies>
+license: EULA.md
 channels:
   # - local  # only in certain situations, like nightly installers where we build napari locally
   - napari/label/bundle_tools  # temporary location of our forks of the constructor stack
   - conda-forge
-specs:
-  - napari
-  - napari-menu  # provides the shortcut configuration for napari
-  - python 3.7  # pinned to the version of the running interpreter, configured in the CI
-  - conda  # we add these to assist in the plugin installations
-  - mamba  # we add these to assist in the plugin installations
-  - pip    # we add these to assist in the plugin installations
+specs: # specs for the 'base'  environment
+  - python   # pinned to the version of the running interpreter, configured in the CI
+  - conda    # we add these to manage different napari versions
+  - mamba    # we add these to manage different napari versions
+  - pip      # we add these to manage different napari versions
+extra_envs:
+  napari-0.4.16: # this is the environment that will actually contain the napari packages
```

Comment:
if we are moving internal versions to 0.0.1, should we update here as well?

Reply:
The env name does contain the napari version it ships. I haven't yet decided (and will probably post an issue asking for comments) about the path naming conventions.

Right it's:

```
napari-installations-0.0.1/envs/napari-0.4.16
\________________________/      \___________/
          base env                napari env
```

So when the user installs/updates to a new version, we would have:

```
```
napari-installations-0.0.1/
+--> envs/
     +--> napari-0.4.16/
     +--> napari-0.4.17/

```